### PR TITLE
ERA-700 serialize record

### PR DESCRIPTION
**Pull request by gaborh-da**

### CODE REVIEW

File: src/main/java/jsonapi/gson/NumericSerializer.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2019, Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package jsonapi.gson;
+
+import com.daml.ledger.javaapi.data.Numeric;
+import com.google.gson.*;
```
Comment (tamaskalcza-da) [inline src/main/java/jsonapi/gson/NumericSerializer.java]: No wildcard imports: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

File: src/test/java/jsonapi/apache/ApacheHttpClientIT.java [hunk 2]
```diff
@@ -33,10 +32,7 @@
 import java.util.List;
 import java.util.Map;
 import jsonapi.JsonApi;
-import jsonapi.gson.ExerciseCommandSerializer;
-import jsonapi.gson.IdentifierSerializer;
-import jsonapi.gson.InstantSerializer;
-import jsonapi.gson.RecordSerializer;
+import jsonapi.gson.*;
```
Comment (tamaskalcza-da) [inline src/test/java/jsonapi/apache/ApacheHttpClientIT.java]: No wildcards.

File: src/test/java/jsonapi/tyrus/TyrusWebSocketClientIT.java [hunk 3]
```diff
@@ -28,10 +27,7 @@
 import jsonapi.JsonApi;
 import jsonapi.events.CreatedEvent;
 import jsonapi.events.Event;
-import jsonapi.gson.CreatedEventDeserializer;
-import jsonapi.gson.IdentifierSerializer;
-import jsonapi.gson.InstantSerializer;
-import jsonapi.gson.WebSocketResponseDeserializer;
+import jsonapi.gson.*;
```
Comment (tamaskalcza-da) [inline src/test/java/jsonapi/tyrus/TyrusWebSocketClientIT.java]: No wildcards.

File: src/main/java/jsonapi/gson/VariantSerializer.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonSerializationContext;
+import com.google.gson.JsonSerializer;
+import java.lang.reflect.Type;
+
+public class VariantSerializer implements JsonSerializer<Variant> {
+
+  @Override
+  public JsonElement serialize(Variant variant, Type type, JsonSerializationContext context) {
```
[the remark this answers is not in the document]
Comment (richardkapolnai-da) [reply src/main/java/jsonapi/gson/VariantSerializer.java]: seems to work, see the tests.

File: src/test/java/jsonapi/SampleJsonSerializerTest.java [hunk 5]
```diff
@@ -40,4 +74,56 @@ public void serializeCurrentTime() {
...
+
+  @Test
+  public void serializeNumeric() {
+    Numeric numeric = new Numeric(BigDecimal.valueOf(12.3));
+    Assert.assertEquals("\"12.3\"", sampleJsonSerializer.apply(numeric));
+  }
+
+  private Date getDate(String date) {
+    long epoch = Instant.parse(date + "T00:00:00Z").getEpochSecond() / 3600 / 24;
+    return new Date((int) epoch);
```
[the remark this answers is not in the document]
Comment (richardkapolnai-da) [reply src/test/java/jsonapi/SampleJsonSerializerTest.java]: this is `com.daml.ledger.javaapi.data.Date`, not `java.util.Date`

**Pull request merged**